media/css/firefox/whatsnew/whatsnew-109-en.scss

```diff
@@ -0,0 +1,56 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import 'includes/base';
+@import 'includes/dark-mode';
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/notification-bar';
+
+.wnp-content-main {
+    .wnp-main-title {
```

Comment:
There should probably be some minimum top padding here (or on the container). The text is right up against the notification in short windows.

![image](https://user-images.githubusercontent.com/205591/212354673-a5376aa9-4fb3-428e-bdbe-ecb75ecd3e13.png)